xds/internal/xdsclient/xdsresource/unmarshal_eds_test.go

```diff
@@ -227,6 +228,42 @@ func (s) TestUnmarshalEndpoints(t *testing.T) {
 				Version: testVersion,
 			},
 		},
+		{
+			name:      "v3 endpoints wrapped",
```

Comment:
Do we need a test for `v2 endpoints wrapped` here like in `unmarshal_cds_test.go`?

Reply:
We don't have v2 tests at all...
And this wrapper is hopefully never used in v2, so this should be OK.